ENIGMAsystem/SHELL/Graphics_Systems/Direct3D9/DX9screen.cpp

```diff
@@ -19,6 +19,7 @@
 #include <cstdio>
 #include "Bridges/General/DX9Context.h"
 #include "Direct3D9Headers.h"
+#include "../General/GSsprite.h"
```

Comment:
Please, do not mention this because I fixed `../` crap in #1238 and will fix the merge conflicts.